docs/reference/contributing/guidelines/license.md

```diff
@@ -57,40 +57,39 @@ In order to clearly reflect the Apache 2.0 license, please create two text files
    See the License for the specific language governing permissions and limitations under the License.
    ```
 
-## When to use the Permissive Binary License
+## When to use PBL
 
-The Permissive Binary License (PBL) is a permissive license based on BSD-3-Clause and designed specifically for binary blobs. It's minimal but covers the basics, including an express patent grant.
+PBL is a permissive license based on BSD-3-Clause and designed specifically for binary blobs. It is minimal but covers the basics, including an express patent grant.
 
-It allows you to share a binary blob and the relevant headers, and allows others to use that binary blob as part of their product - as long as they provide it with all the relevant dependencies and don't modify it or reverse engineer it.
+It allows you to share a binary blob and the relevant headers, and allows others to use that binary blob as part of their product - as long as they provide it with all the relevant dependencies and don't modify or reverse engineer it.
 
-The full text can be found on [mbed.com](https://www.mbed.com/licenses/PBL-1.0).
+The full text is at [mbed.com](https://www.mbed.com/licenses/PBL-1.0).
 
 ## How to apply PBL correctly
 
-In order to clearly reflect the PBL license, please create three text files:
+To clearly reflect the PBL license, please create the following three text files:
 
 - A *LICENSE.md* file with:
 
-<pre>Unless specifically indicated otherwise in a file, files are licensed under the Permissive Binary License,
-as can be found in: LICENSE-permissive-binary-license-1.0.txt</pre>
+    ```
+    Unless specifically indicated otherwise in a file, files are licensed under the Permissive Binary License, as can be found in LICENSE-permissive-binary-license-1.0.txt
+    ```
 
 - The full original [Permissive Binary License 1.0 text](https://www.mbed.com/licenses/PBL-1.0) in *LICENSE-permissive-binary-license-1.0.txt*.
 
 - A *DEPENDENCIES* file with the dependencies that this binary requires to work properly. This is to make sure that third parties integrating the binary in their own distribution are aware that they need to include the relevant dependencies. If your binary does not have any dependencies, the file should state so (that is, say “No dependencies”); don't omit this file.
 
-Each source header should *start with* your copyright line, the SPDX identifier and the PBL header:
+    Each source header should *start with* your copyright line, the SPDX identifier and the PBL header:
```

Comment:
What do you think about "Each source header should *start with* the copyright line, SPDX identifier and PBL header:"?